app/api/v2/models/orders_model.py

```diff
@@ -148,12 +149,16 @@ def cancelled_or_delivered(parcel_id):
         if order:
             return True
 
-    @classmethod
-    def get_by_specific_user(cls, user_id):
+    @staticmethod
+    def get_by_specific_user(user_id):
```

Comment:
This method name is not descriptive enough, it should be improved to `get_orders_by_specific_user`.